add l2C for database + update some deprectated things in dbaas_logs (#595)
ovh/terraform-provider-ovh: website/docs/d/cloud_project_database_log_subscription.html.markdown

---
subcategory : "Managed Databases"
---

# ovh_cloud_project_database_log_subscription (Data Source)

> Use this data source to get information a log subscrition for a cluster associated with a public cloud project.

Reviewer: ```suggestion
Use this data source to get information about a log subscription for a cluster associated with a public cloud project.
```

## Example Usage

```hcl
data "ovh_cloud_project_database_log_subscription" "subscription" {
    service_name = "VVV"
    engine       = "XXX"
    cluster_id   = "YYY"
    id           = "ZZZ"
}

> output "redisuser_commands" {

Reviewer: name `redisuser_commands` seems weird ?

  value = data.ovh_cloud_project_database_log_subscription.subscription.ldp_service_name
}
```

## Argument Reference

The following arguments are supported:

* `service_name` - (Required) The id of the public cloud project. If omitted,
  the `OVH_CLOUD_PROJECT_SERVICE` environment variable is used.

> 

Reviewer: ```suggestion
```

> * `engine` - (Required) The database engine you want to deploy. To get a full list of available engine visit.

Reviewer: ```suggestion
* `engine` - (Required) The database engine for which you want to retrieve a subscription. To get a full list of available engine visit.
```

[public documentation](https://docs.ovh.com/gb/en/publiccloud/databases).

> 

Reviewer: ```suggestion
```

* `cluster_id` - (Required) Cluster ID.

> 

Reviewer: ```suggestion
```
